projects.html

```diff
@@ -42,33 +75,89 @@
       <div class="w-icon-nav-menu"></div>
     </div>
   </div>
-  <div class="about-hero"></div>
-  <div class="about-hero-container">
-    <div class="block-left">
+  <div class="about-hero wf-section"></div>
+  <div class="project_card_container">
+    <div class="left-section">
       <h1 class="primary-heading-3 heading-left"> WDCC Projects</h1>
       <div class="divider-a"></div>
-      <div class="privacy-policy-section-wrapper">
-        <a href="workshop.html" class="link-wrapper privacy-section w-inline-block w-clearfix" data-ix="read-more">
-          <div class="get-started-link"></div>
+      <div class="link_to_project">
+        <a href="#ASPA" class="link-wrapper w-inline-block w-clearfix" data-ix="read-more">
+          <div class="get-started-link-2">- Auckland Student Pool Assoication Event Registeration Website <strong>(ASPA)</strong></div>
+          <div class="hover-line hover-alt"></div>
+          <div class="div-block-27">
+            <div>- Auckland Student Pool Assoication Event Registeration Website </div>
```

Comment:
Association is misspelt here and in line 85 - haven't found anything else though

Reply:
Yupp Thank you Ellen, it is fixed now : )